ext/js/language/ja/japanese-transforms.js

```diff
@@ -662,14 +877,17 @@ export const japaneseTransforms = {
                 suffixInflection('來い', '來る', [], ['vk']),
             ],
         },
-        'imperative negative': {
-            name: 'imperative negative',
-            rules: [
-                suffixInflection('な', '', [], ['v']),
+        'continuative form': {
```

Comment:
I'd like to hear more opinions on this one, since choosing whether we'll refer to this as `masu stem` or `continuative` impacts a lot of the other descriptions. I think it's worth keeping it as `masu stem` since that seems more typical for beginner content

Reply:
This is a hill I will die on to have this change. I don't know what beginner content you're referring to, but not everyone learnt grammar from the same resources that teach 'masu stem.' In fact, I never knew what was that until I used Yomichan. To clarify what is a stem,

書く
語幹＝書
未然形＝か、こ
連用形＝き
終止形＝く
連体形＝く
仮定形＝け
命令形＝け

stem is the unchanging part of the word after inflection, i.e. the common part in its paradigm. Here, clearly the stem is 書, while some other sources would say 'kak' is the stem. How one would find the stem in English or other languages is irrelevant; in Japanese, nobody calls the 'part where you conjugate masu then drop the masu' the 'masu stem'.

Well, I don't want the deinflections to be littered with only linguistically accurate words, but this is a misnomer that I can't close an eye to.

Reply:
I don't think the nomenclature per se is very important, but this is a change to existing user-facing stuff so want to think it through. 

For beginner content, the [tmw guide/grammar](https://learnjapanese.moe/resources/#grammar) recommends [cure dolly](https://docs.google.com/document/d/1XpuXerkGU8waJ4DPDNJA4bGeqOvM-csXjTe57iHARHc/edit), [tae kim](https://gohoneko.neocities.org/grammar/taekim#62%20The%20stem%20of%20verbs), [DoJG](https://gohoneko.neocities.org/grammar/dojg/dojgintermediate#intermediateVmasu). 
Dolly calls it `い-stem`, tae calls it `the stem`. Both mention `masu stem` but don't like it:
> (dolly) In English it's sometimes called the "ます stem", which is not inaccurate because of course
it does, among many, many other things, connect the helper-verb "ます".
I don't like this expression because it stems from the whole ecology where
the so-called ます-form of verbs, which means verbs shifted into the い-stem
and having the "ます" helper verb attached to them, were the base-form of verbs.

> (tae) This is often called the ます-stem in Japanese textbooks but we will call it just the stem because it is used in many more conjugations than just its ます -form. 

Dolly mentions 連用形 a bit, neither mentions `continuative`. DoJG seems to call it `Vmasu`.

All of the options are bad in their own way...